src/components/themed/ThemedModal.tsx

```diff
@@ -1,63 +1,77 @@
 import * as React from 'react'
-import { StyleSheet } from 'react-native'
+import { StyleSheet, ViewStyle } from 'react-native'
 import { AirshipBridge, AirshipModal } from 'react-native-airship'
+import { BlurView } from 'rn-id-blurview'
 
 import { fixSides } from '../../util/sides'
 import { useTheme } from '../services/ThemeContext'
+import { ModalFooter, ModalScrollArea } from './ModalParts'
 
 interface Props<T> {
   bridge: AirshipBridge<T>
   children?: React.ReactNode
+  onCancel: () => void
 
   // Control over the content area:
+  closeButton?: boolean
+  flexDirection?: ViewStyle['flexDirection']
+  justifyContent?: ViewStyle['justifyContent']
   paddingRem?: number[] | number
 
-  // Adds a yellow warning border:
-  warning?: boolean
+  // Scroll area with a fade
+  scroll?: boolean
 
-  onCancel: () => void
+  // For determining the appropriate background blur color
+  isDark?: boolean
```

Comment:
Remove this prop, and make this flag 100% theme-based.

When the theme uses light colors, we need to tweak the blur so it looks good. Since whitelabel apps use light themes, and since even dev mode can switch to light-mode at run-time, `useTheme` is the only way to get this value correctly.

If there is no `isDark` flag in the login-ui theme, then we need to add it.